services/blockchain-connector/shared/sdk/events.js

```diff
@@ -68,8 +75,19 @@ const getEventsByHeightFormatted = async (height) => {
 	return formattedEvents;
 };
 
+const verifyClientConnection = async () => {
+	setInterval(() => isClientConnected = false, BLOCK_TIME * numOfBlocksReset);
+	setInterval(async () => {
+		if (!isClientConnected) {
+			logger.info('Re-instantiating API client.');
+			await instantiateClient(true);
+		}
+	}, BLOCK_TIME * numOfBlocksVerify);
```

Comment:
If we recreate a connection that is not broken, could it create any issue for ongoing node calls? Also, would it be better to reuse [getApiClient](https://github.com/LiskHQ/lisk-service/blob/51b074adc43117d70d514fc622f4ce7b8d09c402/services/blockchain-connector/shared/sdk/client.js#L85) logic, which already checks for broken connection and recreates it? Also, we do not need the `numOfBlocksReset` logic in this case.

```suggestion
	setInterval(getApiClient, BLOCK_TIME * numOfBlocksVerify);
```